go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/nutanix-cloud-native/cluster-api-provider-nutanix
 
 go 1.22.1
 
+toolchain go1.22.2
```

Comment:
Note: devbox did not have 1.22.2 hence had to make it this way

Reply:
Why does the version needs to be bumped?

Reply:
https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/pull/512#pullrequestreview-2001724387
else caren's govulnerability check fails due to http/net version

Reply:
This toolchain version wouldn't affect CAREN afaik as it is local to working on the module here.